dist/wrapper.js

```diff
@@ -0,0 +1,173 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.wrapper = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
I want it to be in the package on NPM, because I don't (yet) want the bundling to happen at runtime. And since it's in the NPM package on NPM, it seemed natural to include it in Git, too. It'll help us detect unexpected differences in the bundling process.

If I was writing a Node.js package using future ECMAScript syntax or some other incompatible compiles-to-JavaScript language, then I definitely _wouldn't_ include the build output in Git. I guess I'm just being inconsistent for no reason I can sufficiently articulate? /shrug